_explore/scripts/get_repos_dependencies.py

```diff
@@ -0,0 +1,59 @@
+from scraper.github import queryManager as qm
+from os import environ as env
+
+ghDataDir = env.get("GITHUB_DATA", "../github-data")
+datfilepath = "%s/labRepos_Dependencies.json" % ghDataDir
+queryPath = "../queries/repo-dependencies.gql"
```

Comment:
Minor nitpicking detail, but could we change the `repo-dependencies.gql` filename to `repo-Dependencies.gql`? Just so that the naming scheme matching the other query files.

Reply:
I think the actual path here still needs to be changed to match the re-named file.